demisto_sdk/commands/common/tests/base_validator_test.py

```diff
@@ -1,6 +1,23 @@
+import os
+
+from demisto_sdk.commands.common.git_tools import git_path
 from demisto_sdk.commands.common.hook_validations.base_validator import \
     BaseValidator
 
+DEPRECATED_IGNORE_ERRORS_DEFAULT_LIST = ['IN100', 'IN101', 'IN102', 'IN103', 'IN104', 'IN105', 'IN106', 'IN107',
```

Comment:
don't think it's a good thing, we will have to update this all the time

Reply:
done